### ISSUE TITLE
[A11y Maintenance Audit]: Pre-need (Form 40-10007)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#47352]
## Auditor(s)

Josh Kim

## Automated

- [x] Run axe checks on every page or unique state (required)
- [x] Run WAVE on every page if time allows. WAVE is more visual than axe or axe-coconut, but offers some excellent ways to identify nested headings and HTML5 landmark tags.

## Manual

- [x] Color contrast checks (recommend using whocanuse.com for beyond compliance reports)
- [x] Color blindness checks
- [ ] Zoom layouts to 400% and inspect them for readability. If layouts break at 400%, I will start reducing them until they become stable, and log the zoom ratio when things started breaking.
- [ ] Keyboard navigation

## Screen Reader

- [ ] Firefox and NVDA
- [ ] Edge and JAWS
- [x] iOS Safari and VoiceOver (Mobile)
- [x] Safari and Voiceover (Desktop)
- [ ] Chrome and Talkback

## Staging or Production Link

[Staging link](https://staging.va.gov/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/introduction)
[Production link](https://www.va.gov/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/introduction)

## Test Users

```bash
1. Unsigned in user (start without signing in)
2.
3.
...
```

**Comment by MichelleDieudonne** [comment]
This should be fixed by work completed in epic https://jira.devops.va.gov/browse/MBMS-53401
